release-1.5/release-notes-draft.md

```diff
@@ -0,0 +1,104 @@
+
+- Improved Federation Support
+  - Daemonsets
+  - Deployments
+  - ConfigMaps
+- Simplified Cluster Deployment
+  - `kubeadm`
+  - `kubefed`
+  - HA Setup for Master
+- Node Robustness
+  - Windows Server Container support
+  - CRI implementation
+  - `kubelet` API supports authentication and authorization
+
+## Features
+
+Features for this release were tracked via the use of the [kubernetes/features](https://github.com/kubernetes/features) issues repo.  Each Feature issue is owned by a Special Interest Group from [kubernetes/community](https://github.com/kubernetes/community)
+
+- **Apps**
+  - [stable] When replica sets cannot create pods, they will now report detail via the API about the underlying reason ([kubernetes/features#120](https://github.com/kubernetes/features/issues/120))
+  - [stable] `kubectl apply` is now able to delete resources you no longer need with `--prune` ([kubernetes/features#128](https://github.com/kubernetes/features/issues/128))
+  - [beta] Deployments that cannot make progress in rolling out the newest version will now indicate via the API they are blocked ([docs](https://deploy-preview-1699--kubernetes-io-vnext-staging.netlify.com/docs/user-guide/deployments/#failed-deployment)) ([kubernetes/features#122](https://github.com/kubernetes/features/issues/122))
+  - [beta] StatefulSets allow workloads that require persistent identity or per-instance storage to be created and managed on Kubernetes. ([docs](https://deploy-preview-1719--kubernetes-io-vnext-staging.netlify.com/docs/concepts/controllers/statefulsets/)) ([kubernetes/features#137](https://github.com/kubernetes/features/issues/137))
+  - [beta] In order to preserve safety guarantees the cluster no longer force deletes pods on un-responsive nodes and users are now warned if they try to force delete pods via the CLI. ([dos](https://deploy-preview-1774--kubernetes-io-vnext-staging.netlify.com/docs/tasks/manage-stateful-set/scale-stateful-set/)) ([kubernetes/features#119](https://github.com/kubernetes/features/issues/119))
+- **Auth**
+  - [alpha] Further polishing of the Role-based access control alpha API including a default set of cluster roles. ([docs](http://kubernetes.io/docs/admin/authorization/)) ([kubernetes/features#2](https://github.com/kubernetes/features/issues/2))
+  - [beta] Added ability to authenticate/authorize access to the Kubelet API ([docs](https://deploy-preview-1684--kubernetes-io-vnext-staging.netlify.com/docs/admin/kubelet-authentication-authorization/)) ([kubernetes/features#89](https://github.com/kubernetes/features/issues/89))
+- **AWS**
+  - [stable] Roles should appear in kubectl get nodes ([kubernetes/features#113](https://github.com/kubernetes/features/issues/113))
+- **Cluster Lifecycle**
+  - [alpha] Use kubeadm to make it easy to get a cluster running. ([docs](https://deploy-preview-1684--kubernetes-io-vnext-staging.netlify.com/docs/admin/kubeadm/)) ([kubernetes/features#11](https://github.com/kubernetes/features/issues/11))
```

Comment:
Maybe: [alpha] Improved UX and usability for the `kubeadm` binary that makes it easy to get a new cluster running

The link should be `docs/getting-started-guides/kubeadm/`